.vscode/launch.json

```diff
@@ -4,9 +4,23 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "LCFS - pytests",
+      "type": "debugpy",
+      "request": "launch",
+      "cwd": "${workspaceFolder}/backend",
+      "module": "poetry",
+      "python": "${workspaceFolder}/backend/venv/bin/python3",
+      "args": [
+        "run",
+        "pytest",
+        "-v",
+        "lcfs/tests/test_users.py::test_get_users_with_single_sortOrder"
```

Comment:
i think this is left over?

Reply:
yes, I removed specific test case, but this can be use to debug pytests in vscode.